app/controllers/admin/tutorials_controller.rb

```diff
@@ -29,6 +29,13 @@ def update
     redirect_to edit_admin_tutorial_path(tutorial)
   end
 
+  def destroy
+    tutorial = Tutorial.find(params[:id])
+    tutorial.destroy
+    redirect_to admin_dashboard_path
+    # binding.pry
```

Comment:
Looks good! This binding.pry can be removed at any time.

Reply:
Oops! Thanks for catching that.  I should starting searching the whole for prys before pushing the branch.